packages/smooth_app/lib/generic_lib/bottom_sheets/smooth_bottom_sheet.dart

```diff
@@ -318,6 +318,7 @@ class SmoothModalSheet extends StatelessWidget {
     Color? headerForegroundColor,
     this.bodyPadding,
     this.expandBody = false,
+    this.staticContent = false, // New parameter
```

Comment:
How come is it `false` by default?
Most of the time it's stupid static content like "blablabla click-ok", isn't it?